src/features/embalm/stepContent/hooks/useCreateSarcohpagus.ts

```diff
@@ -170,15 +173,16 @@ export function useCreateSarcophagus() {
     uploadFile
   ]);
 
-  const resetLocalEmbalmerState = useCallback(() => {
-    setArchaeologistShards([] as ArchaeologistEncryptedShard[]);
-    setEncryptedShardsTxId('');
-    setSarcophagusPayloadTxId('');
-    setArchaeologistSignatures(new Map<string, string>([]));
-    setNegotiationTimestamp(0);
-    setOuterPrivateKey('');
-    setOuterPublicKey('');
-  }, []);
+  // TODO -- re-enable once figure out state issue at end of createSarcophagus
+  // const resetLocalEmbalmerState = useCallback(() => {
+  //   setArchaeologistShards([] as ArchaeologistEncryptedShard[]);
+  //   setEncryptedShardsTxId('');
+  //   setSarcophagusPayloadTxId('');
+  //   setArchaeologistSignatures(new Map<string, string>([]));
+  //   setNegotiationTimestamp(0);
+  //   setOuterPrivateKey('');
+  //   setOuterPublicKey('');
+  // }, []);
```

Comment:
We will want this method, but this may be causing issues when it is cleared out. Need to do more testing.